modules/services/openssh.nix

```diff
@@ -0,0 +1,137 @@
+# Parts from nixpkgs/nixos/modules/services/networking/ssh/sshd.nix
+# MIT Licensed. Copyright (c) 2003-2022 Eelco Dolstra and the Nixpkgs/NixOS contributors
+
+{ pkgs, lib, config, ... }:
```

Comment:
Please have a look at the other files for examples of our licence header comment.